BingMaps/spatial-data-services/public-data-sources/pointsofinterest.md

```diff
@@ -14,7 +14,7 @@ ms.service: "bing-maps"
 > [!NOTE]
 > **Bing Maps Spatial Data Service feature retirement – Points of Interest**
 >
-> The Bing Maps **Spatial Data Service – Points of Interest** data source is now deprecated and will be retired on 6/30/2026. To avoid service disruptions, modify your application by 6/30/2026. Azure Maps offers this capability and provides security, privacy and compliance enhancements. To get Point of Interest data we recommend leveraging the Azure Maps [Get Search POI](/rest/api/maps/search/get-search-poi) API. Resources to aid in your migration:
+> The Bing Maps **Spatial Data Service – Points of Interest** data source is deprecated and will be retired on June 30, 2026. To avoid service disruptions, please update your application by this date. Azure Maps offers this capability with enhanced security, privacy, and compliance features. For Point of Interest data, we recommend using the Azure Maps [Get Search POI](/rest/api/maps/search/get-search-poi) API. Resources to aid in your migration:
```

Comment:
"The Bing Maps Spatial Data Service – Points of Interest data source is deprecated and will be retired on June 30, 2026. Azure Maps offers this capability with enhanced security, privacy, and compliance features. For Point of Interest data, we recommend updating your application to use the Azure Maps [Get Search POI](https://github.com/rest/api/maps/search/get-search-poi) API before that date to avoid service disruptions. We have the following resources available to aid in your migration:"

Reply:
So, your problem is with this sentence? "To avoid service disruptions, please update your application by this date."

Reply:
No, the statement is still there. Two sentences were merged into one complete sentence and the final sentence fragment was changed to align with what you wrote for MIO.